### ISSUE TITLE
Add supported operations, protocols and security guarantees to README

### ISSUE THREAD

**Issue by kamathhrishi** [OpenMined/SyMPC#248]
## Where?
Add a table of operations we support, the protocol supporting it, a number of parties supported and security guarantees to the README. Something like this 

| Operation       | Protocol         | Security Guarantee | Number of parties |
| ------------- |:-------------:| -----:| -----:|
| Addition     | SPDZ| Semi honest | 2+ |
| Subtraction      | FSS    |  Semi-honest | 2+ |
| Addition     | Falcon  |  Malicious | 2+ |
| Subtraction   | Falcon  |  Malicious | 2+ |

Also add Neural Network layers supported 

| Layer     | Protocol         | Security Guarantee | Number of parties |
| ------------- |:-------------:| -----:| -----:|
| Maxpool   | Falcon,SPDZ,FSS | Semi honest | 2+ |
| Linear     | Falcon, SPDZ,FSS    |  Semi-honest | 2+ |
| conv2d     | Falcon  |  Malicious | 2+ |

(Above operation information are just examples, might not be correct)

**Comment by danielorihuela** [comment]
I have a couple of questions. 

1. Should I count comparison operators as operations, and add them to the table?
2. I do not know how to easily check with neural network layers are supported with which protocols. I see that SPDZ has a list like _EXPECTED_OPS = {"mul", "matmul", "conv2d", "conv_transpose2d"}_. But I do not know how to "extract" more relations.

**Comment by kamathhrishi** [comment]
Hey @danielorihuela this are the list of operations supported: add, sub, division, xor, Le, ge, eq, group these into one (mult, matmul ,conv2d, conv2d_tranpose), sigmoid, relu, softmax, reciprocal and argmax, 

The supported network layers are only Linear and Conv2d for now.

**Comment by kamathhrishi** [comment]
I am think maybe we should have this information in a separate Markdown file instead of having it in Readme with a link to the readme to supported protocols, ops and layers.

**Comment by kamathhrishi** [comment]
I was also think you could make it three tables: 1) Supported protocols 2) Support operations 3) Support torch neural network layers.

**Comment by danielorihuela** [comment]
I tried to make the tables. I do not know if the operations are correct. Furthermore, I do not know how to relate network layers to MPC protocols. I do not see any hint on the code that points me that Linear works with SPDZ for example.
